tests/test_read.py

```diff
@@ -83,10 +85,18 @@ def test_btl_is_dataframe(btl):
     assert not btl.empty
 
 
-def test_btl_with_dup_cols(btl_duplicate_header_name):
-    assert all(
-        col in btl_duplicate_header_name.columns for col in ["Bottle", "Bottle_"]
-    )
+def test_btl_with_dup_cols(btl_as_stream):
+    assert all(col in btl_as_stream.columns for col in ["Bottle", "Bottle_"])
+
+
+def test_btl_as_stringio(btl_as_stream):
+    assert isinstance(btl_as_stream, pd.DataFrame)
+    assert not btl_as_stream.empty
+
+
+def test_btl_as_stringio_without_name(btl_as_stream):
+    assert isinstance(btl_as_stream, pd.DataFrame)
+    assert not btl_as_stream.empty
```

Comment:
`btl_as_stream` is from a stream and without a name. so you don't need the `test_btl_as_stringio` and `test_btl_as_stringio_without_name`, only one would suffice.